README.md

```diff
@@ -1,2 +1,18 @@
-[![Review Assignment Due Date](https://classroom.github.com/assets/deadline-readme-button-24ddc0f5d75046c5622901739e7c5dd533143b0c8e959d652212380cedb1ea36.svg)](https://classroom.github.com/a/6BOvYMwN)
-# AssignmentTemplate
\ No newline at end of file
+# ay2324s1-course-assessment-g47
+ay2324s1-course-assessment-g47 created by GitHub Classroom
+
+Steps to run:
```

Comment:
I think is better if the steps are more detailed. For example, when to change directory and where to run the command at which directory in the steps.

ie. Do npm run install-all in the main directory etc

Reply:
Noted, I have done the relevant changes as suggested 👍